journald/modules.json.cmake-in

```diff
@@ -0,0 +1,30 @@
+{
+  "libraries": [
+    {
+      "file": "@CMAKE_CURRENT_BINARY_DIR@/@CMAKE_BUILD_TYPE@/libjournaldlogger.dll",
```

Comment:
How does one configure and build modules so that `CMAKE_BUILD_TYPE` is substituted correctly?

(I tried to perform clean build with cmake as described in README.md - looks like `CMAKE_BUILD_TYPE` is empty at the time when substitution is performed).

Reply:
`DCMAKE_BUILD_TYPE` needs to be passed during configuration (updated README to reflect this).